### PR TITLE
Balance input test case

### PR DESCRIPTION
**Pull request by Ekaanth**
Test case for balances component

### CODE REVIEW

File: src/ui/components/Balances.test.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import React from 'react';
+import Balances from '@ui/components/Balances';
+import {create} from 'react-test-renderer';
+import {render, within} from 'src/testUtils';
+
+jest.doMock('src/ui/components/Balances', () => {
```
Comment (silva-fj) [inline src/ui/components/Balances.test.tsx]: You should not mock the function/component under test.

File: src/ui/components/Balances.test.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  return {
+    BalancesMockComponent: jest.fn(() => <Balances balance={accountBalanceDetails} />),
+  };
+});
+
+const {BalancesMockComponent} = require('src/ui/components/Balances');
+
+describe('Balances component', () => {
+  it('should render the prop component correctly', async () => {
+    const {getByTestId} = render(<BalancesMockComponent />);
```
Comment (silva-fj) [inline src/ui/components/Balances.test.tsx]: As I mentioned last week, query by `testID` should be avoided if possible, you should try to query for the things that the user will see

**Review by silva-fj — COMMENTED** [review]
Don't spend much time in this kind of tests, as I said they don't provide much value

**Comment by Ekaanth** [comment]
I am closing this PR since this PR doesn't add any benefit to the app test cases.

**Pull request closed without merging**